org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeCoordinatorHandler.java

```diff
@@ -425,6 +434,13 @@ private void startZigBeeNetwork() {
         }, 1, TimeUnit.SECONDS);
     }
 
+    @Override
+    public void handleRemoval() {
+        // this flag is used in dispose() in order to do some special stuff when the bridge has been removed
```

Comment:
Suggestion: Be more explicit, and write "to remove the serialized network info" instead of "do some special stuff".

Reply:
OK